modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -21,180 +21,181 @@
     <test name="is-tests-default-configuration" preserve-order="true" parallel="false" group-by-instances="true">
```

Comment:
Seems you have added the commented `testng.xml`. Can you remove this from the PR?